packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -25,15 +24,40 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   const filterTerminatedCards = (cards) =>
     cards.filter((card) => card.status !== CardStateType.TERMINATED)
 
+  const getEligibleAccounts = function* (cardId) {
+    try {
+      const data = yield call(api.getDCEligibleAccounts, cardId)
+      yield put(A.setEligibleAccounts(data))
```

Comment:
Do you think it's a good idea to make this a remote?

Reply:
For now I don't see it necessary, but in the next PR I will be working showing this list in a Flyout and will see if it is needed to change this to a remote type. With the current logic is not necessary.

Reply:
alrighty